model.py

```diff
@@ -0,0 +1,66 @@
+import torch.nn as nn
+import torch
+import copy
+from modules.attention import MultiHeadedAttention
+from modules.fnn import PositionwiseFeedForward
+from modules.positional_encoding import PositionalEncoding
+from modules.encoderdecoder import EncoderDecoder, Generator, subsequent_mask
+from modules.decoder import Decoder, DecoderLayer
+from modules.encoder import Encoder, EncoderLayer
+from modules.embeddings import Embeddings
+
+
+def make_model(
+    src_vocab, tgt_vocab, n=6, d_model=512, d_ff=2048, h=8, dropout=0.1
+):
```

Comment:
```suggestion
def make_model(
    src_vocab,
    tgt_vocab,
    n=6,
    d_model=512,
    d_ff=2048,
    h=8,
    dropout=0.1,
):
```

Always break multi-argument definitions into multiple lines and include the "magic trailing comma" after the last one. (Also use type hints :))

Reply:
Srryyy, I will clean it tomorrow, that is a work in progress (not rly legible or working yet)

Reply:
No worries, I just wanted to finish the repo setup and left some comments out of habit 😅

Reply:
okay, thank you!! : )